docs/src/sections/social_aspects.adoc

```diff
@@ -1,3 +1,46 @@
-=== Społeczne aspekty projektu
+=== Społeczne i etyczne aspekty projektu
 
-TODO
+Kryptowaluty są technologią rewolucjonizującą rynek finansowy. Patrząc na wartość giełdową niektórych z nich,
+można domniemywać, że mogą stanąć się alternatywnym środkiem płatniczym wobec tradycyjnych walut. Tworzony system
+może wspomóc obrót Bitcoin'em, czyli najdroższą i najpopularniejszą z kryptowalut. Kryptowaluty z założenia nie
+podlegają kontroli żadnych instytucji, gdyż wykorzystywana przez nie technologia Blockchain jest niczym innym jak
+zdecentralizowaną bazą danych. Dowód pracy oraz system imisji środków poprzez wykopywanie nowych bloków z
+transakcjami sprawia, że jest to niezwykle bezpieczny środek płatniczy. Atak na całą sieć jest praktycznie
+niemożliwy poprzez ilość mocy obliczeniowej aktualnie z nią związanej. Hakerom bardziej opłaca się wykopywać bloki
+w normalny sposób, niż marnować prąd na próby ataków siłowych. Jedynym zagrożeniem jest utrata bądź kradzież kluczy
+prywatnych użytkownika, ale tak samo, jak w przypadku tradycyjnych pieniędzy użytkownik powinien zadbać o ich
+bezpieczeństwo samemu, bądź ze wsparciem specjalnych rozwiązań oferowanych przez firmy. Bitcoin jest również wolny od
+inflacji. Emisja środków odbywa się poprzez nagradzanie osób, którym uda się wykopać blok z transakcjami. Nie ma innej
+możliwości włączenia środków do sieci. Istnieje odgórny limit ilości Bitcoin'ów, które kiedykolwiek zostaną wyemitowane
+i wynosi 21 milionów. Żaden bank ani żadna organizacja nie będzie mogła manipulować wartością i ilością pieniędzy w
+obiegu, co czyni je stabilnym środkiem płatniczym.footnote:[Odnosząc się do wartości dóbr w relacji do Bitcoina, a nie
+do wartości giełdowej kryptowalut, która bywa bardzo zmienna] Transakcje są jedynie podpisane kluczami prywatnymi,
+a nie danymi osobowymi jak w przypadku normalnych walut. Zapewnia to pełną anonimowość transakcji.
+
+Powyższe zalety Bitcoina, jak i kryptowalut w ogóle rodzą również wiele zagrożeń. Anonimowość transakcji może okazać
+się idealnym rozwiązaniem dla wszelkiego rodzaju przestępców. Można sobie wyobrazić sytuację, w której kryptowaluty
+wykorzystywane są do nielegalnych płatności za broń czy narkotyki. Jest to poważny argument przeciwko cyfrowym
+pieniądzom, aczkolwiek jest to cena, jaką trzeba zapłacić za pełną anonimowość. Zyskują zarówno osoby z dobrymi, jak
+i złymi intencjami. Kolejnym problemem jest decentralizacja systemu. Nie ma żadnej instytucji nadzorującej, a co
+za tym idzie, nie ma do kogo się zgłosić w przypadku problemów. W przypadku niezrozumiałej utraty środków nie ma
```

Comment:
napisałbym raczej : 
"W przypadku utraty środków wynikającej np z niewiedzy czy też pomyłki..."
bo chyba to miałeś na myśli